-------------------------------------------------------------------------------
Test set: com.zion.school.SchoolApplicationTests
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 7.195 s <<< FAILURE! - in com.zion.school.SchoolApplicationTests
contextLoads  Time elapsed: 0.003 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaConfiguration': Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.springframework.boot.autoconfigure.jdbc.DataSourceInitializerInvoker': Invocation of init method failed; nested exception is org.springframework.jdbc.datasource.init.ScriptStatementFailedException: Failed to execute SQL script statement #2 of class path resource [db/schema.sql]: CREATE TABLE SIBLING_INFORMATION ( ID INTEGER NOT NULL PRIMARY KEY, NAME VARCHAR(256) NOT NULL, AGE INTEGER NOT NULL ); nested exception is org.postgresql.util.PSQLException: ERROR: relation "sibling_information" already exists
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.springframework.boot.autoconfigure.jdbc.DataSourceInitializerInvoker': Invocation of init method failed; nested exception is org.springframework.jdbc.datasource.init.ScriptStatementFailedException: Failed to execute SQL script statement #2 of class path resource [db/schema.sql]: CREATE TABLE SIBLING_INFORMATION ( ID INTEGER NOT NULL PRIMARY KEY, NAME VARCHAR(256) NOT NULL, AGE INTEGER NOT NULL ); nested exception is org.postgresql.util.PSQLException: ERROR: relation "sibling_information" already exists
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.springframework.boot.autoconfigure.jdbc.DataSourceInitializerInvoker': Invocation of init method failed; nested exception is org.springframework.jdbc.datasource.init.ScriptStatementFailedException: Failed to execute SQL script statement #2 of class path resource [db/schema.sql]: CREATE TABLE SIBLING_INFORMATION ( ID INTEGER NOT NULL PRIMARY KEY, NAME VARCHAR(256) NOT NULL, AGE INTEGER NOT NULL ); nested exception is org.postgresql.util.PSQLException: ERROR: relation "sibling_information" already exists
Caused by: org.springframework.jdbc.datasource.init.ScriptStatementFailedException: Failed to execute SQL script statement #2 of class path resource [db/schema.sql]: CREATE TABLE SIBLING_INFORMATION ( ID INTEGER NOT NULL PRIMARY KEY, NAME VARCHAR(256) NOT NULL, AGE INTEGER NOT NULL ); nested exception is org.postgresql.util.PSQLException: ERROR: relation "sibling_information" already exists
Caused by: org.postgresql.util.PSQLException: ERROR: relation "sibling_information" already exists

